src/LibChorus.TestUtilities/RepositorySetup.cs

```diff
@@ -219,9 +217,11 @@ public static void MakeRepositoryForTest(string newRepositoryPath, string userId
 			hg.SetUserNameInIni(userId,  progress);
 		}
 
-		public static string ProjectName
+		public string ProjectName { get; private set; }
+
+		public static string ProjectNameForTest
 		{
-			get { return "ไก่ projéct"; } //nb: important that it have a space, as this helps catch failure to enclose in quotes
```

Comment:
Yes, I decided that seeing as this was the option that broke, I'd switch to having CP1252 being the greater test coverage.  I've also added tests in the utf8tests to cover either option.  Previously these tests were defective; 'CreateOrUseExisting' was not suffcient to test, and RecoverFromInteruptedTransaction which showed the failure in client apps was not tested anywhere else in Chorus.  So, now we should have at least one test that works for each of unicode and cp1252.